client/src/main/resources/templates/js/htmlreporter.js

```diff
@@ -16,36 +16,36 @@
 $(document).ready(function () {
 
   "use strict";
```

Comment:
why not just change this to `'use strict';` instead and keep the use of `'` everywhere in tact?